tests/unit/archive_base_test.py

```diff
@@ -111,11 +112,13 @@ async def test_download_file(mocker, file_data):
     # Initialize MockArchiver class
     archiver = MockArchiver(None)
 
-    session_mock = mocker.MagicMock(name="session_mock")
+    session_mock = mocker.AsyncMock(name="session_mock")
     archiver.session = session_mock
 
     # Set return value
-    session_mock.get.return_value.__aenter__.return_value.read.return_value = file_data
+    response_mock = mocker.AsyncMock()
```

Comment:
I had to restructure the mocks since I took stuff out of the context manager for the retries.